### PR TITLE
updated abnormal condition

### PR DESCRIPTION
**Pull request by Sakshi-dell**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] unit test is successful
- [x] Checked the condition, by deleting export from OneFS, the volume is abnormal along with the error

### CODE REVIEW

**Comment by prablr79** [comment]
@Sakshi-dell please do update description and defect details here in this PR.

File: service/controller.go [hunk 1]
```diff
@@ -1699,24 +1700,31 @@ func (s *service) ControllerGetVolume(ctx context.Context,
 
 	//check if volume exists
 	if !isiConfig.isiSvc.IsVolumeExistent(ctx, isiPath, volName, "") {
+		abnormal = true
+		message = fmt.Sprintf("volume does not exists at this path %v", isiPath)
```
Comment (bpjain2004) [inline service/controller.go:1704]: why are we not returning from here when the volume is abnormal. Does it make sense to proceed with other checks ?

Comment (Sakshi-dell) [reply service/controller.go:1704]: we are not checking anything else, the second if condition will be executed only if volume is not abnormal
It's just we have a common return statement for both these abnormal cases

Comment (randeepsharma) [reply service/controller.go:1704]: change exists to exist. (volume does not exist at this path %v)

**Review by randeepsharma — APPROVED** [review]
LGTM

**Review by bpjain2004 — APPROVED** [review]

**Pull request merged**